src/biophysics.jl

```diff
@@ -370,4 +370,123 @@ function evap(
   J_evap = uconvert(u"g/s",J_evap)
 
   (Q_evap = Q_evap, J_evap = J_evap, J_resp = J_resp, J_cut = J_cut, J_eyes = J_eyes)
+end
+
+function resp(
+    T_x = (23+273.15)K,
+    mass = 0.04kg,
+    Q_metab = 0.01241022W,
+    O2_ext_ref = 20,
+    pant = 1,
+    rq = 0.8,
+    T_air = (20+273.15)K,
+    rh = 50,
+    P_atmos = 101325Pa,
+    pctO2 = 20.95,
+    pctCO2 = 0.03,
+    pctN2 = 79.02)
+  # C     COMPUTES RESPIRATORY HEAT AND WATER LOSS VIA MASS FLOW THROUGH THE LUNGS,
```

Comment:
The capitalized comments give me the uneasy feeling that OTIS is yelling at me from 1964

Reply:
fixed